src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/OasysService.kt

```diff
@@ -179,6 +168,37 @@ class OasysService(
     }
   }
 
+  fun getAssessmentDateInfo(prisonNumber: String): OasysAssessmentDateInfo {
```

Comment:
This method ends up calling Oasys twice for the same data - can we change this to only call it once? Also it looks for any OPEN status - I thought it would only chose ones with a higher id than the current layer3?